releases/release-1.20/release-notes-draft.md

```diff
@@ -1,178 +1,258 @@
-# Release notes for v1.20.0-beta.2
+# Release notes for v1.20.0-rc.0
 
 [Documentation](https://docs.k8s.io/docs/home)
 # Changelog since v1.19.0
 
+## What's New (Major Themes)
+
+### Dockershim deprecation
+Docker as an underlying runtime is being deprecated. Docker-produced images will continue to work in your cluster with all runtimes, as they always have.
+The Kubernetes community [has written a blog post about this in detail](https://blog.k8s.io/2020/12/02/dont-panic-kubernetes-and-docker/) with [a dedicated FAQ page for it](https://blog.k8s.io/2020/12/02/dockershim-faq/).
+
+### External credential provider for client-go
+With this release, client-go credential plugins can now be passed in the current cluster information via the KUBERNETES_EXEC_INFO environment variable. Learn more about this on [client-go credential plugins documentation](https://docs.k8s.io/reference/access-authn-authz/authentication/#client-go-credential-plugins/).
+
+### CronJob controller v2 is available through feature gate
+An alternative implementation of CronJob controller is now available as an alpha feature in this release, which has expermiental performance improvement by using informers instead of polling. While this will be the default behavior in the future, you can [try them in this release through a feature gate](https://docs.k8s.io/concepts/workloads/controllers/cron-jobs/).
+
+### PID Limits graduates to General Availability
+After being enabled-by-default for a year, SIG Node graduates PID Limits to GA on both `SupportNodePidsLimit` (node-to-pod PID isolation) and `SupportPodPidsLimit` (ability to limit PIDs per pod).
+
+### API Priority and Fairness graduates to Beta
+Initially introduced in 1.18, Kubernetes 1.20 now enables API Priority and Fairness (APF) by default. This allows `kube-apiserver` to [categorize incoming requests by priority levels](https://docs.k8s.io/concepts/cluster-administration/flow-control/).
+
+### IPv4/IPv6 dual-stack
+IPv4/IPv6 dual-stack has been reimplemented for 1.20 to support dual-stack Services, based on user and community feedback. If your cluster has dual-stack enabled, you can create Services which can use IPv4, IPv6, or both, and you can change this setting for existing Services. Details are available in updated [IPv4/IPv6 dual-stack docs](https://docs.k8s.io/concepts/services-networking/dual-stack/), which cover the nuanced array of options.
+
+We expect this implementation to progress to beta and GA in coming releases, so we’re eager to have you comment about your dual-stack experiences in [#k8s-dual-stack](https://kubernetes.slack.com/messages/k8s-dual-stack) or in [enhancements #563](https://features.k8s.io/563). 
```

Comment:
@bridgetkromhout 👍🏼 👎🏼 ?

Reply:
On further review, I'd change "We expect this implementation" to "We expect this alpha implementation" just to be extra-clear?

Reply:
hmm I'm not a trained technical writer, but to me "this alpha implementation" seem to imply that there won't be any changes until they graduate -- is that accurate?

alternatively, I was thinking more along the lines of "We expect his implementation approach to progress from alpha to beta and GA"

thoughts?

Reply:
Ah, sure! Okay, what if we changed "We expect this implementation to progress to beta and GA in coming releases" to:
```
We expect this implementation to progress from alpha to beta and GA in coming releases
```

basically I want the term alpha in there somewhere, for people who are looking for it.

Reply:
If you don't think it works there, then a possible other place it could go is:
"IPv4/IPv6 dual-stack has been reimplemented for"

changing to "IPv4/IPv6 dual-stack has been reimplemented as alpha for"

Reply:
I like the first one! WIP...